dapr/proto/runtime/v1/dapr.proto

```diff
@@ -654,4 +664,34 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowReference  = 1;
+  string WorkflowType = 2;
+  string InstanceID = 3;
+  string WorkflowComponent = 4; 
+}
+
+message GetWorkflowResponse {
+  WorkflowReference WorkflowReference  = 1;
+  google.protobuf.Timestamp StartTime = 2;
+  map<string, string> Metadata = 3;
+}
+
+message StartWorkflowRequest {
+  WorkflowReference WorkflowReference = 1;
+  string WorkflowComponent = 2;
+  string WorkflowName = 3;
+  map<string, string> Options = 4;
+  google.protobuf.Any Parameters = 5;   
```

Comment:
`Parameters` as a name for this field is a little confusing because using a plural often implies some kind of collection type, which isn't the case here. My recommendation would be to call this `StartParameter` (single) or `Input`, which is the terminology we use for the built-in workflow engine.